src/lib/glocal/glocal.ts

```diff
@@ -43,7 +43,7 @@ export function setup({ run, toolName, depName, filename }: SetupInput): void {
   // use envar to boost perf, skip costly detection work
   if (!process.env.GLOBAL_LOCAL_HANDOFF) {
     log.trace('execLayout start')
-    const execLayout = detectExecLayout({ depName })
+    const execLayout = detectExecLayout({ depName, scriptPath: filename })
```

Comment:
Doesn't this mean that we're no longer using the scriptPath default of `process.argv[0]`? Why? Remove the default if no longer needed?

Reply:
Yeah so, we haven't talked about it yet, but it turns out that on Windows, argv[1] is **always** an absolute path to the js file and never the .bin/<depname> path. Therefore, in order to "normalize" paths betweeen the platforms, I've decided to always pass the __filename.
This is also the reason why I had to implement the logic to resolve the script path from the package.json `bin` property. Since we never get the information of the .bin/<depname> executable on windows, and since argv[1] is also an absolute path to the script, the only way to compare the executed script path with the local script path is to resolve the path of the script from the package.json.

Reply:
> is always an absolute path to the js file and never the .bin/ path.

As was the case on macOS (or Linux, one of the two, can't remember). That's what [this](https://github.com/graphql-nexus/nexus/pull/1184/files#diff-1bef84169496baa6927a46188ac4519aL177) was dealing with for example.

> Since we never get the information of the .bin/ executable on windows

What do we need that info for, though?

The ultimate goal has always been to compare the executed scripts of current process vs current project.

I had previously worked with `.bin` and symlinks because that's what argv[1] pointed to before. I suspect that the symlink was good for performance because of less IO to trace the project tool path.

What I'm hearing now is that in Windows, this work is essentially already done for us. It sounds like more reason, not less, to use `argv[1]`?